log-router/templates/service.yaml

```diff
@@ -15,6 +15,7 @@ metadata:
     prometheus.io/scrape: "true"
     prometheus.io/port: "24231"
   name: {{ template "fluentd-router.fullname" . }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
Not sure it is needed. I think helm injects the namespace - none of the stable charts define it, for example https://github.com/helm/charts/blob/master/stable/kubernetes-dashboard/templates/deployment.yaml

Reply:
Oh, I got confused and you are right. Then I'm removing this part of the PR